matrix-client/matrix-client.cabal

```diff
@@ -49,8 +49,8 @@ common common-options
 common lib-depends
   build-depends:       SHA                    ^>= 1.6
                      , base64                 >= 0.4.2 && < 0.5
-                     , bytestring             >= 0.11.3 && < 0.12
-                     , containers             >= 0.6.5 && < 0.7
+                     , bytestring             >= 0.11.3 && < 0.13
```

Comment:
I guess we could also put `< 1`, I would not expect our usage of bytestring to break anytime soon.